### PR TITLE
AdminToy.List

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

File: EXILED/Exiled.API/Features/Map.cs [hunk 1]
```diff
@@ -130,7 +125,7 @@ public static Scp939AmnesticCloudInstance AmnesticCloudPrefab
         /// <summary>
         /// Gets all <see cref="AdminToy"/> objects.
         /// </summary>
-        public static ReadOnlyCollection<AdminToy> Toys { get; } = ToysValue.AsReadOnly();
+        public static ReadOnlyCollection<AdminToy> Toys => AdminToy.BaseToAdminToy.Values.ToList().AsReadOnly();
```
Comment (IRacle1) [inline EXILED/Exiled.API/Features/Map.cs:128]: are you sure to make that list so inefficiently?

`Dictianary.ValueCollection` is inherited from `ReadOnlyCollection`, so we can just

```suggestion
        public static ReadOnlyCollection<AdminToy> Toys => AdminToy.BaseToAdminToy.Values;
```

Comment (IRacle1) [reply EXILED/Exiled.API/Features/Map.cs:128]: Oh i missed that it is `ReadOnlyCollection`, not `IReadOnlyCollection` interface, we cant do it more memory efficient😭

File: EXILED/Exiled.API/Features/Toys/AdminToy.cs [hunk 2]
```diff
@@ -33,9 +40,14 @@ internal AdminToy(AdminToyBase toyAdminToyBase, AdminToyType type)
...
             ToyType = type;
 
-            Map.ToysValue.Add(this);
+            BaseToAdminToy.Add(toyAdminToyBase, this);
         }
 
+        /// <summary>
+        /// Gets a list of all <see cref="AdminToy"/>'s on the server.
+        /// </summary>
+        public static IEnumerable<AdminToy> List => BaseToAdminToy.Values;
```
Comment (IRacle1) [inline EXILED/Exiled.API/Features/Toys/AdminToy.cs:49]: can be `IReadOnlyCollection`

```suggestion
        public static IReadOnlyCollection<AdminToy> List => BaseToAdminToy.Values;
```

File: EXILED/Exiled.API/Features/Map.cs (hunk 1 above)
Comment (louis1706) [reply EXILED/Exiled.API/Features/Map.cs:128]: yup anyway this will be deleted for Exiled9

**Review by Jesus-QC — APPROVED** [review]

**Review by Misfiy — APPROVED** [review]

**Pull request merged**